docs/source/framework/pytorch_integration/layers_database.rst

```diff
@@ -0,0 +1,364 @@
+ML Layers database
```

Comment:
Can we delete this file?

The main reason I am asking is because the list of layers mostly comes from C++ land with slight variations and risks making this the source of truth for PyTorch users. Making this the source of truth comes with all the issues mentioned previously by both @ftynse  and myself:
1. copy-pasta synchronization issues (we now have many of these layers in 3 places)
2. not the actual source of truth since the source of truth is always C++ and people's programs will break at some unexpected point in the future
3. the syntax is not TC but makes the ```{...}``` templating by string replace hack a first class citizen of the language

Reply:
nope, we can't delete this. It's pretty important and useful doc entry. 

Let's wait out for the Issue #74  to be resolved and then we can circulate at it.